packages/dds/matrix/src/permutationvector.ts

```diff
@@ -208,12 +208,7 @@ export class PermutationVector extends Client {
 		return this.getPosition(segment) + offset!;
 	}
 
-	public handleToPosition(handle: Handle, localSeq = this.getCollabWindow().localSeq) {
-		assert(
-			localSeq <= this.getCollabWindow().localSeq,
-			0x028 /* "'localSeq' for op being resubmitted must be <= the 'localSeq' of the last submitted op." */,
-		);
-
+	public handleToSegment(handle: Handle) {
 		// TODO: In theory, the MergeTree should be able to map the (position, refSeq, localSeq) from
```

Comment:
'handleToPosition(..)' is a pre-existing method that implemented the linear scan to map a handle to a position, computing the segment + offset as an intermediate step.

I factored out the intermediate step as 'handleToSegment(..)' for reuse above.